.github/workflows/canary-test-config/action.yaml

```diff
@@ -13,13 +13,24 @@ runs:
       with:
         go-version: 1.19
 
-    - name: setup minikube
-      uses: manusa/actions-setup-minikube@v2.7.2
-      with:
-        minikube version: "v1.27.0"
-        kubernetes version: "v1.25.0"
-        start args: --memory 6g --cpus=2 --addons ingress
-        github token: ${{ inputs.github-token }}
+    - name: Setup Minikube
+      shell: bash --noprofile --norc -eo pipefail -x {0}
+      run: |
+        sudo apt update
+        sudo apt install -y conntrack socat
+        curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube_latest_amd64.deb
+        sudo dpkg -i minikube_latest_amd64.deb
+        rm -f minikube_latest_amd64.deb
+        curl -LO https://github.com/Mirantis/cri-dockerd/releases/download/v0.3.0/cri-dockerd_0.3.0.3-0.ubuntu-focal_amd64.deb
+        sudo dpkg -i cri-dockerd_0.3.0.3-0.ubuntu-focal_amd64.deb
+        rm -f cri-dockerd_0.3.0.3-0.ubuntu-focal_amd64.deb
+        VERSION="v1.26.0"
```

Comment:
How about moving this version to to the top of the action so it's easy to update later

Reply:
actually removed this version as this is not the Kubernetes version  and can cause confusion